dlc-node/src/main/scala/org/bitcoins/dlc/node/DLCNode.scala

```diff
@@ -79,4 +73,32 @@ case class DLCNode(wallet: DLCWalletApi)(implicit
       accept
     }
   }
+
+  override def sendDLCOffer(
+      peerAddress: InetSocketAddress,
+      incomingOfferHash: Sha256Digest): Future[Unit] = {
+    for {
+      offer <- wallet.findIncomingDLCOffer(incomingOfferHash)
```

Comment:
This is not what we want to be doing. We want to create a new one that is not in IncomingOffers.